.github/workflows/ef-tests.yml

```diff
@@ -0,0 +1,79 @@
+name: EF Tests
+
+on:
+  pull_request:
+    branches:
+      - main
+
+permissions: read-all
+
+jobs:
+  ef-tests:
+    # trunk-ignore(actionlint/runner-label)
+    runs-on: ubuntu-latest-32-cores
+    steps:
+      # Install packages
+      - name: install packages
+        run: sudo apt-get install curl
+
+      - name: Install Foundry
```

Comment:
why do we need foundry?

Reply:
Nice catch, we needed it with last approach, we no more need it.

Reply:
resolved.